reference/enterprise-api-ref/cmdb-api.markdown

```diff
@@ -0,0 +1,317 @@
+---
+layout: default
+title: CMDB API
+published: true
+tags: [reference, enterprise, API, CMDB, classes, variables]
+---
+
+Configuration management database API enables you to manage classes and variables for specific hosts.
+
+## List CMDB 
+
+You can see a list of stored hosts-specific configurations
+
+**URI:** https://hub.cfengine.com/cmdb
+
+**Method:** GET
+
+**Parameters:**
+* **fromEpoch** *(integer)*
+  Returns configurations with epoch value greater than set in the filter.
+  Epoch is the sequence number of the latest CMDB change. In every API list request, you
+  might see `most_recent_cmdb_change` property on the meta section, which contains the maximum
```

Comment:
why "might"? Shouldn't it either be "will" or not? 

s/on the meta/in the meta/